src/cmds/dslJson.ts

```diff
@@ -0,0 +1,37 @@
+import { readFile } from "fs/promises";
+import { Options as YargsOptions } from "yargs";
+
+import checkDSL from "../validator";
+import friendlySyntaxToApiSyntax from "../transformer";
+
+interface CommandArgs {
+  dslFile: string;
+}
+
+exports.command = "dslToJSON <dslFile>";
```

Comment:
This currently allows `@openfga/syntax-transformer dslToJSON <fileName>`

I think namespacing it by: `transform` allows for more flexibility in the future.

E.g. `@openfga/syntax-transformer transform --from=dsl <fileName>`

If the from is not passed, we try to get the from from the passed in file type

`to` is always the opposite

Later, we can add `--to` if we see a need to, but there is no need now.

Reply:
@matthewpereira any thoughts? This is just for transformation at the moment

Reply:
Updated with new syntax.

New syntax is 
```
npx @openfga/syntax-transformer transform --from=json --inputFile=test.json
npx @openfga/syntax-transformer transform --from=dsl --inputFile=test.dsl
```

Help menu shows
```
npx @openfga/syntax-transformer transform --help
syntax-transformer transform

Transform

Options:
  --version    Show version number                                     [boolean]
  --help       Show help                                               [boolean]
  --from       whether we want to transform from dsl or json
                                             [required] [choices: "dsl", "json"]
  --inputFile  Configuration file. It must be in DSL syntax. [string] [required]
```